runtime-modules/forum/src/mock.rs

```diff
@@ -514,6 +467,12 @@ pub fn migration_not_done_config() -> GenesisConfig<Runtime> {
     create_genesis_config(false)
 }
 
+pub fn generate_hash() -> <Runtime as system::Trait>::Hash {
+    let hash = <Runtime as system::Trait>::Hash::random();
```

Comment:
I presume this generates a random hash during the tests? If so then I think we should avoid this, any form of non-determinism in testing invariably leads to hard to detect problems later.

Reply:
Random hash generation was replaced by a better (deterministic) mock.